tests/set-labels/empty-list/simple/.expected/results.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
 exitCode: 1
 items:
   - image: gcr.io/kpt-fn/set-labels:unstable
-    stderr: '[error] : failed to configure function: function config must be a ConfigMap or SetLabelConfig'
+    stderr: '[error] : failed to configure function: function config must be a `ConfigMap` or `SetLabels`'
```

Comment:
it's "functionConfig" not "function config". The former is a specific enttity, the latter is two generic words.